### DISCUSSION TITLE
General discussion of Investing Algorithm Framework

### DISCUSSION THREAD

**Discussion by MDUYN** [coding-kitties/investing-algorithm-framework#211] [General]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, you can comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amustapha** [comment]
I started a couple of discussions and would like to get your opinion on those as well.

Also, I saw a couple of things on the project board which I think might be helpful if you can add a few descriptions and I can try finding one to pickup and get started with.

**Reply by MDUYN** [reply]
Hey @amustapha, sorry for getting back to you so late. I will update the board for you this week. Thanks for reaching out!

**Reply by MDUYN** [reply]
@amustapha there is now a contributing guideline for the project: https://github.com/coding-kitties/investing-algorithm-framework/blob/main/docs/CONTRIBUTING.md. If you would like to pick up a task that is part of the overall development please pickup a task from the project board that can be found [here](https://github.com/orgs/coding-kitties/projects/3). If you need any help with the tasks, feel free to start a discussion [here](https://github.com/coding-kitties/investing-algorithm-framework/discussions).
